pytest.ini

```diff
@@ -0,0 +1,8 @@
+[pytest]
```

Comment:
I know @c24t had some reservations about using pytest. We were supposed to discuss this after the alpha release.

FWIW I really like pytest and wholeheartedly support using pytest and pytest functionality.

Reply:
> I know @c24t had some reservations about using pytest.

Hello @c24t, can you please clarify what kind of reservations you had?
Thank you!

> We were supposed to discuss this after the alpha release.

I think nothing prevents us from discussing it a bit earlier.

Also, I found a related issue: #28.

Reply:
My biggest complaints about pytest have to do with fixtures.  Fixtures with side effects, autouse fixtures, and the `monkeypatch` fixture in particular have the potential to turn any test suite into a horrifying mess. Using fixtures as arguments in tests makes working with tests difficult outside of pytest, and adds a lot of magic. I don't mean to say it's not possible to write nice tests with pytest, just that the library encourages some pretty bad (IMHO) behavior.

When we considered adding it before it wasn't clear that there were any features of pytest that we couldn't get just as easily from the built in unittest library. It may be that the extra complexity is justified if pytest makes writing and running tests easier. I still don't see a strong reason for using pytest in this PR, but I may be missing something here. What makes `pytest --cov` better than `coverage run`?

Reply:
Hello @toumorokoshi and @c24t,

On the `SIG meeting 2019-09-12` we discussed migration to `pytest` and, if I recall correctly, we found that most developers are fine with it or have no opinion. Unfortunately, we didn't write out a corresponding decision in the [OT Python SIG Notes](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs).
So my question is, can we resolve this thread now and move forward with `pytest` or is it necessary to have an additional discussion?

Thank you!

Reply:
@toumorokoshi To actually dismiss your review you have to expand the "Changes requested" box at the bottom of the PR, and then click "..." -> "Dismiss" next to your review there. Re-requesting your review won't actually dismiss it (I ran into that too once, it's not the most intuitive UI).